introductory/src/buttonComponent.jsx

```diff
@@ -0,0 +1,13 @@
+export default function Button(props){
+    const {id, clickHandle} = props;
```

Comment:
**Optional**
you can add these prop as parameters since there are not many instead of destructuring them outside